e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -953,385 +946,245 @@ describe("scenarios > embedding > full app", () => {
-            ...ordersCardDetails,
-            type: cardType,
-            collection_id: FIRST_COLLECTION_ID,
-          };
-
-          cy.signInAsAdmin();
-          H.createQuestion(cardDetails);
-          cy.log("grant `nocollection` user access to `First collection`");
-          cy.updateCollectionGraph({
-            [ALL_USERS_GROUP]: { [FIRST_COLLECTION_ID]: "read" },
-          });
-
-          cy.signIn("nocollection");
-          startNewEmbeddingQuestion({ isMultiStageDataPicker: true });
-          selectCard({
-            cardName: cardDetails.name,
-            cardType,
-            collectionNames: ["First collection"],
-          });
-          clickOnDataSource(ordersCardDetails.name);
-          verifyCardSelected({
-            cardName: cardDetails.name,
-            collectionName: "First collection",
-          });
-        });
-
-        it("should select a data source when there is no access to the immediate parent collection", () => {
-          const cardDetails = {
-            ...ordersCardDetails,
-            type: cardType,
-            collection_id: THIRD_COLLECTION_ID,
-          };
-
-          cy.signInAsAdmin();
-          H.createQuestion(cardDetails);
-          cy.updateCollectionGraph({
-            [ALL_USERS_GROUP]: {
-              [FIRST_COLLECTION_ID]: "read",
-              [THIRD_COLLECTION_ID]: "read",
-            },
-          });
-
-          cy.signIn("nocollection");
-          startNewEmbeddingQuestion({ isMultiStageDataPicker: true });
-          selectCard({
-            cardName: cardDetails.name,
-            cardType,
-            collectionNames: ["Third collection"],
-          });
-          clickOnDataSource(ordersCardDetails.name);
-          verifyCardSelected({
-            cardName: cardDetails.name,
-            collectionName: "Third collection",
-          });
-        });
-
-        it("should be able to join a card when the data source is a table", () => {
-          const cardDetails = {
-            ...ordersCardDetails,
-            type: cardType,
-            collection_id: FIRST_COLLECTION_ID,
-          };
-          H.createQuestion(cardDetails);
-          startNewEmbeddingQuestion({ isMultiStageDataPicker: true });
-          selectTable({
-            tableName: "Products",
-          });
-          H.getNotebookStep("data").button("Join data").click();
-          H.popover().within(() => {
-            cy.icon("chevronleft").click();
-            cy.icon("chevronleft").click();
-          });
-          selectCard({
-            cardName: cardDetails.name,
-            cardType,
-            collectionNames: ["First collection"],
-          });
-          clickOnJoinDataSource(cardDetails.name);
-          verifyCardSelected({
-            cardName: cardDetails.name,
-            collectionName: "First collection",
-          });
-        });
+    describe("question", () => {
+      beforeEach(() => {
+        cy.intercept({
```

Comment:
This interception is needed to ensure that when we revert our code this test would fail, without it, the test would pass because `Saved questions` and `Metrics` are rendered after `databases` response has been received.